csharp/elasticbeanstalk/elasticbeanstalk-environment/src/ElasticbeanstalkEnvironment/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Amazon.CDK;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ElasticbeanstalkEnvironment
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var app = new App(null);
```

Comment:
See `null` comment

Reply:
This is how the sample-app is initialized, should we change the template?